metaflow/datastore/flow_datastore.py

```diff
@@ -93,6 +93,8 @@ def get_latest_task_datastores(
         allow_not_done : bool, optional
             If True, returns the latest attempt of a task even if that attempt
             wasn't marked as done, by default False
+        mode : str, optional
```

Comment:
nit: this is incorrect in terms of type hint, it should be: "mode : str, default "r"" or something like that.